scene/3d/path.cpp

```diff
@@ -43,6 +43,15 @@ void Path::_curve_changed() {
 	if (is_inside_tree()) {
 		emit_signal("curve_changed");
 	}
+
+	// update the configuration warnings of all children of type OrientedPathFollows
+	if (is_inside_tree()) {
+		for (int i = 0; i < get_child_count(); i++) {
+			if (OrientedPathFollow *child = Object::cast_to<OrientedPathFollow>(get_child(i))) {
```

Comment:
I guess using `PathFollow` here would be better (or adding a similar if for `PathFollow`)

Reply:
No, this is intended, because only `OrientedPathFollow` could actually change its behaviour based on the changed `up_vectors_enabled`. A plain `PathFollow` does not care about the up vectors.

Reply:
Ah, sorry, didn't notice what it is about.